.github/workflows/docker_build.yml

```diff
@@ -22,18 +22,19 @@ jobs:
       fail-fast: False
       matrix:
         TF_VERSION:
-          - 0.12
-          - 0.13
-          - 0.14
-          - latest
+          - "0.12"
+          - "0.13"
+          - "0.14"
+          - "1.0"
```

Comment:
Currently only Terragrunt version 0.31 supports Terraform 1.0, which I am not seeing in this list

https://github.com/gruntwork-io/terragrunt/releases/tag/v0.31.0

Reply:
the change was only from double/float to string, no new version was added here ;)